packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -11,6 +11,10 @@ import { compose } from '@wordpress/compose';
  */
 import { getPasteEventData } from '../../utils/get-paste-event-data';
 
+function isNativePasteEvent( event ) {
+	return event.type === 'paste' && event.target.tagName === 'INPUT';
```

Comment:
Is this the right check here, won't we have the same issues for contenteditables, textareas....
I feel the intent here is more: 

 - trigger the native paste behavior if the target is inside the editor canvas 

I believe the main issue here is that there's React event bubbling happening instead of native DOM event bubbling triggering the onPaste handler when it shouldn't. 

One option to solve this is to avoid the React handlers or to check the `event.target` inside the handlers.

Reply:
I was thinking it was more of an isolated case hence treating the symptoms rather than the cause. Will look into your suggestions.

Reply:
Updated the PR.